src/acc/field_summary.cpp

```diff
@@ -82,9 +82,21 @@ void field_summary(global_variables &globals, parallel_ &parallel) {
     double *pressure = field.pressure.data;
     double *xvel0 = field.xvel0.data;
     double *yvel0 = field.yvel0.data;
+    
+    if (globals.profiler_on) {
+      globals.profiler.summary += timer() - kernel_time;
+      kernel_time = timer();
+    }
+
+#pragma acc enter data copyin(vol, mass, ie, ke, press)
```

Comment:
I think we should keep this in mind when we do analysis, but OpenMP/OpenACC will have slightly higher `host_to_device` time while other model will have higher `device_to_host` timers in these reduction loops.